tcti/tcti_socket.c

```diff
@@ -144,34 +129,14 @@ TSS2_RC SocketSendTpmCommand(
     }
 
 #ifdef DEBUG
```

Comment:
We should not need these #ifdef DEBUG anymore.

Reply:
no "DEBUG" used anymore